app/decorators/news_decorator.rb

```diff
@@ -5,8 +5,11 @@ def short_lead
     "#{model.body.first(50)}..."
   end
 
-  def lead 
-    "#{model.body.first(250)}..."
+  def generate_lead 
+    @sentences = ActionController::Base.helpers.strip_tags(model.body).split('.')
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.